client/src/reducers/local-storage.reducer.ts

```diff
@@ -1,48 +1,48 @@
+import { createReducer } from '@reduxjs/toolkit';
 import { HelpCard } from '../actions/help-card.actions';
 import { Persist } from '../actions/persist.actions';
 import { User } from '../actions/user.actions';
+import { UserActions } from '../slices/user.slice';
 import { Dict } from '../utility/typescript-utils';
 import { getLoggedInState } from '../utility/user-utils';
 
 // We need to be careful about what goes into local storage due to security concerns. So make sure not to add anything
 // that is clearly identifying of groups, units, or individuals, such as PII/PHI data, group name, etc.
-
-export interface LocalStorageState extends Record<string, unknown>{
+export interface LocalStorageState extends Record<string, any>{
   orgId?: number
   hideHelpCard?: Dict<boolean>
 }
 
 export const localStorageInitialState: LocalStorageState = {};
 
-export function localStorageReducer(state = localStorageInitialState, action: any): LocalStorageState {
-  switch (action.type) {
-    case User.Actions.Register.type: {
+export const localStorageReducer = createReducer(localStorageInitialState, builder => {
+  builder.addCase(UserActions.refresh.fulfilled, (state, action) => {
+    const { userData, localStorage } = action.payload;
+    const loggedInState = getLoggedInState(userData, localStorage);
+    return {
+      ...state,
+      orgId: loggedInState.activeRole?.role.org?.id,
+    };
+  })
+    .addCase(UserActions.register.fulfilled, (state, action) => {
       const { userData, localStorage } = (action as User.Actions.Register).payload;
       const loggedInState = getLoggedInState(userData, localStorage);
       return {
         ...state,
         orgId: loggedInState.activeRole?.role.org?.id,
       };
-    }
-    case User.Actions.Refresh.type: {
-      const { userData, localStorage } = (action as User.Actions.Refresh).payload;
-      const loggedInState = getLoggedInState(userData, localStorage);
-      return {
-        ...state,
-        orgId: loggedInState.activeRole?.role.org?.id,
-      };
-    }
-    case User.Actions.Logout.type: {
+    })
+    .addCase(UserActions.logout.type, () => {
       return localStorageInitialState;
-    }
-    case User.Actions.ChangeOrg.type: {
+    })
+    .addCase(UserActions.changeOrg.type, (state, action) => {
       const { orgId } = (action as User.Actions.ChangeOrg).payload;
```

Comment:
It looks like we can do this sort of thing to stay within the redux-toolkit style...

```suggestion
    .addCase(UserActions.changeOrg, (state, { payload }) => {
      const { orgId } = payload;
```

Apparently we never need to use the `type` property, since the action creators already override the `toString()` function.